Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodeGroupWindow.cpp

```diff
@@ -139,10 +134,16 @@ namespace EMStudio
     void NodeGroupRenameWindow::Accepted()
     {
         // Execute the command
-        AZStd::string commandString, outResult;
+        AZStd::string outResult;
         const AZStd::string convertedNewName = FromQtString(mLineEdit->text());
-        commandString = AZStd::string::format("AnimGraphAdjustNodeGroup -animGraphID %i -name \"%s\" -newName \"%s\"", mAnimGraph->GetID(), mNodeGroup.c_str(), convertedNewName.c_str());
-        if (GetCommandManager()->ExecuteCommand(commandString.c_str(), outResult) == false)
+        auto* command = aznew CommandSystem::CommandAnimGraphAdjustNodeGroup(
+            GetCommandManager()->FindCommand("AnimGraphAdjustNodeGroup"),
+            mAnimGraph->GetID(),
+            /*name=*/mNodeGroup,
+            /*visible=*/AZStd::nullopt,
+            /*newName=*/convertedNewName
```

Comment:
Nit: Mismatch between this place and others, space between the comment and code.

Reply:
I fixed all these to be consistent.